overview.md

```diff
@@ -0,0 +1,58 @@
+Anbox Cloud provides a rich software stack to enable you to run Android in the cloud for all kinds of different uses cases. This page intends to give you an overview over the available variants and internal components which make Anbox Cloud.
+
+## Variants
+
+Anbox Cloud exists in two different variants which serve different purposes:
+
+**Anbox Cloud Appliance**
+
+The Anbox Cloud Appliance is a self-contained, single-machine deployment variant of Anbox Cloud. It uses a sensible standard configuration to hide the complexity and flexibility of Juju. In this way, the Anbox Cloud Appliance allows simple and fast development and prototyping.
```

Comment:
Updated this description. Are we okay with this wording?

Reply:
Sounds good to me. Do we maybe want to link to Juju here to provide more context if needed?